public/app/plugins/datasource/prometheus/specs/language_provider.test.ts

```diff
@@ -79,12 +80,13 @@ describe('Language completion provider', () => {
       expect(result.suggestions).toMatchObject([
         {
           items: [
-            { label: '1m' },
-            { label: '5m' },
-            { label: '10m' },
-            { label: '30m' },
-            { label: '1h' },
-            { label: '1d' },
+            { label: '$__interval', sortText: '$__interval' }, // TODO: figure out why this row and sortText is needed
```

Comment:
I imagine that something has changed, does anyone know what happened here?

Reply:
This row is fine, it helps users discover that the automatic step interval can be used for the range vectors, but in practice it's too fine. I prefer it if we sort `$__interval` after the normal options. The sort text prop helps with this. `sortText` is needed in general because e.g.,  1m < 1h semantically, but not lexicographically.